packages/@webex/plugin-meetings/src/member/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface IExternalRoles {
+  cohost: boolean;
+  moderator: boolean;
+  presenter: boolean;
+}
+
+export enum ServerRoles {
+  Cohost = 'COHOST',
+  Moderator = 'MODERATOR',
+  Presenter = 'PRESENTER',
```

Comment:
question: Is there a reason why these are first-letter capitalized?  I've seen mostly all caps in the SDK, and some all lower.